vizro-core/docs/pages/tutorials/first-dashboard.md

```diff
@@ -81,7 +84,7 @@ Paste the following example into a Notebook cell, run it, and view the generated
 
     [Dashboard]: ../../assets/user_guides/dashboard/dashboard.png
 
-
+<!-- vale off -->
 ### 4. Explore further
-
-You are now ready to explore Vizro further, by working through the [Explore Vizro tutorial](../tutorials/explore-components.md) or by consulting the [how-to guides](../user-guides/dashboard.md).
+<!-- vale on -->
+You are now ready to explore Vizro further, by working through the [Explore Vizro tutorial](./explore-components.md) or by consulting the [how-to guides](../user-guides/dashboard.md).
```

Comment:
Shall we add "" or similar for page titles?

```suggestion
You are now ready to explore Vizro further, by working through the ["Explore Vizro" tutorial](./explore-components.md) or by consulting the [how-to guides](../user-guides/dashboard.md).
```

Reply:
This is such a difficult one to decide as the links aren't very consistent -- one's a title the other is a group of pages. Let's try your idea!